test/cppsuite/test_harness/debug_utils.h

```diff
@@ -41,10 +43,14 @@ static int64_t _trace_level = 0;
 
 /* Used to print out traces for debugging purpose. */
 static void
-debug_info(const std::string &str, int64_t trace_threshold, int64_t trace_level)
+debug_info(const std::string &str, int64_t trace_level, int64_t trace_type)
```

Comment:
I know this PR is kinda turning into a "fix debug logging" PR but I'm gonna ask for more changes. Can we rename this to `debug_print` and can we not pass in the global trace_level? In the future if we find a reason to pass in the global trace level I'd be happy to spin out a `debug_print_custom_trace` function that gets called from `debug_print`. But as I don't think any of our callers will be supplying a custom trace level for now we don't need that.